fastlane/fastlane
# [supply] automatically rescue changesNotSentForReview errors

### Checklist

### Motivation and Context


Resolves #18670 (again...)

### Description


Added 'rescue_changes_not_sent_for_review' option to supply to specify whether or not to retry committing an edit using information gathered from an error message. 


### Testing Steps


If an app/track is in "rejected" state, then you need to submit the app with changesNotSentForReview=true, otherwise you should send without specifying changesNotSentForReview (or setting it to false)
